docs/adr/lts-strategy.md

```diff
@@ -0,0 +1,176 @@
+- Some other related discussions:
+  - https://github.com/expressjs/discussions/issues/196
+  - https://github.com/expressjs/discussions/issues/172
+  - https://github.com/expressjs/discussions/issues/67
+- TC Slack Discussion: https://openjs-foundation.slack.com/archives/C06KMCETHG9/p1724863852681889
+
+### Package/Repository Types
+
+We maintain four primary types of packages/repos:
+
+1. `express`: The minimal http framework
+2. `express` dependencies: Libraries which are used directly or transitively within `express` and also by others outside
+   of the project
+3. Middleware: Libraries for use *with* `express` but not installed with it directly
+4. Others: Tools, documentation, websites, resources, etc.
+
+While much of this proposal is targeted specifically at `express`, this terminology will be used to differentiate where
+we are not only talking about `express`.
+
+## Decision
+
+The Express project will have a well defined Long Term Support (LTS) strategy and schedule based on the following
+requirements:
+
+- `express` major versions will go through three supported phases: Current, Active, & Maintenance
+  - `CURRENT`: A new major version is designated as `CURRENT` upon release. It is available but not the `latest` version
+    on npm for a minimum of 3 months.
+  - `ACTIVE`: After the minimum 3 month period and the TC agrees it is stable and secure, the `ACTIVE` version is
+    tagged `latest` on npm for a minimum of 12 months.
+  - `MAINTENANCE`: When a new major version becomes `ACTIVE`, the previous major version enters `MAINTENANCE` for 12 months.
+- After the `MAINTENANCE` time has ended the major version is considered `EOL` and is unsupported.
+- During the 12 month `ACTIVE` period, a new major version may be released but *not* until at least 6 months have passed
+  since the `CURRENT` version became `ACTIVE`.
+- Users are required to follow the head/latest of each major release line for support with all packages (`express`,
+  dependencies, middleware, & tools/other)
+- If we have no necessary breaking changes, we will not release a new major version. Applies to `express`,
+  dependencies, middleware, & tools/other.
+- Dropping old Node.js versions *alone* is not enough of a breaking change to release a new major version. Applies to
+  `express`, dependencies, middleware, & tools/other.
+- `express` version 4.x is considered a special case. It will receive a longer `MAINTENENCE` phase of 18 months or until
+  the TC agrees we are save cutting support.
+- `express` dependency packages will follow the same timeline and support dates for the `express` version which they are
+  included with.
+- Middleware packages should follow the timeline and support dates for the `express` versions they are compatible, but
+  should make reasonable and informed decisions which can deviate if necessary.
+- All other packages are suggested to define an appropriate strategy for support.
+
+### Node.js Version Support
```

Comment:
It is questionable if this section belongs in this ADR. IMO they are separate but have some connections. We do not want our support/release plan to conflict too much with Node.js so that we avoid supporting too many EOL Node.js versions. But other than that, maybe we just remove this section for now and make a separate "node.js version support strategy"?

Reply:
Public docs however would benefit of having both explained in one place.

Reply:
Ah, good point and yes I agree fully. I will add that to the section about public docs.